src/lib/components/tabs/QTab.svelte

```diff
@@ -1,86 +1,55 @@
 <script lang="ts">
-  import { onMount } from "svelte";
+  import { getContext, hasContext } from "svelte";
   import { QIcon } from "$lib";
+  import QContext from "$lib/classes/QContext.svelte";
   import { ripple } from "$lib/helpers";
-  import {
-    isActivationKey,
-    isArrowKey,
-    getDirection,
-    getClosestFocusableSibling,
-    isTabKey,
-    getClosestFocusableBlock,
-  } from "$lib/utils";
-  import type { Direction } from "$lib/utils";
+  import { getClosestFocusableBlock, getClosestFocusableSibling } from "$lib/utils/dom";
+  import type { Direction } from "$lib/utils/events";
+  import { getDirection, isActivationKey, isArrowKey, isTabKey } from "$lib/utils/events";
```

Comment:
Kinda weird it allows you to put the non-`type` import there, I'll have to check that later.